src/groupeddataframe/groupeddataframe.jl

```diff
@@ -27,6 +37,11 @@ function Base.getproperty(gd::GroupedDataFrame, f::Symbol)
             gd.idx, gd.starts, gd.ends = compute_indices(gd.groups, gd.ngroups)
         end
         return getfield(gd, f)::Vector{Int}
+    elseif f == :keymap
+        if getfield(gd, f) === nothing
```

Comment:
Just for consistency with other fields:
```suggestion
        # Keymap is computed lazily the first time it is accessed
        if getfield(gd, f) === nothing
```